treeherder/etl/buildbot.py

```diff
@@ -939,9 +942,11 @@
     "Marionette Framework Unit Tests": "Mn",
     "Marionette Framework Unit Tests e10s": "Mn-e10s",
     "Marionette WebAPI Tests": "Mnw",
-    "Media Tests": "Mt",
     "Android x86 Test Set": "S",
     "Android x86 Test Combos": "Sets",
+    "VideoPuppeteer": "VP",
+    "Media Tests MSE Video Playback": "b-m",
+    "Media Tests MSE YouTube Playback": "b-y",
```

Comment:
Yeah... :) We need some way to distinguish the tests that run on **b**uildbot per commit and the same ones that run on our Jenkins instance for each release. I'm open to suggestions.